### PR TITLE
Item card

### PR DESCRIPTION
**Pull request by NamburiYaswanth**
image

### CODE REVIEW

**Review by tulasikurete — APPROVED** [review]

**Review by NagaDurga16 — APPROVED** [review]

**Comment by Namahsivaya7** [comment]
please add space between images and their titles

**Comment by Namahsivaya7** [comment]
where is mobile view

**Comment by Namahsivaya7** [comment]
page width must be 100% but it is morethan that

**Review by arajajyothibabu — DISMISSED** [review]

File: src/app/checkoutpage/page.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { LoadingOutlined, SmileOutlined, SolutionOutlined, UserOutlined } from '@ant-design/icons';
+import { Steps,Typography } from 'antd';
+
+const CheckOut: React.FC = () => (
```
Comment (arajajyothibabu) [inline src/app/checkoutpage/page.tsx]: Remove this file and changes from this PR

**Review by arajajyothibabu — CHANGES_REQUESTED** [review]

**Review by arajajyothibabu — APPROVED** [review]
LGTM

**Pull request merged**